modules/evaluation/src/main/PlayerAssessment.scala

```diff
@@ -135,7 +153,7 @@ object PlayerAssessment:
         case _                                   => NotCheating
 
       if (flags.suspiciousHoldAlert) assessment
-      else if (~game.wonBy(color)) assessment
+      else if (~game.wonBy(color)) antichessCorrectedGameAssessment(assessment, pov, flags)
```

Comment:
looks like this bypasses the following `if` statements for all non-antichess games?